core/chains/evm/logpoller/orm_test.go

```diff
@@ -745,7 +745,7 @@ func TestORM_IndexedLogs(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, 3, len(lgs))
 
-	lgs, err = o1.FilteredLogs(ctx, rangeFilter(1, 1, 3), limiter, "")
+	lgs, err = o1.FilteredLogs(ctx, rangeFilter(1, 1, 3).Expressions, limiter, "")
```

Comment:
You can update rangeFilter to return a slice of Expressions instead of KeyFilter

Reply:
Addressed: [`7550e2b` (#14109)](https://github.com/smartcontractkit/chainlink/pull/14109/commits/7550e2b7721736099d277be946922cd73a962133)